README.md

```diff
@@ -45,11 +45,11 @@ It will ask you the name you want to pick interactively.
 cargo install cargo-generate
 git clone https://github.com/copper-project/copper-rs
 cd copper-rs/templates
-cargo nunew [path_where_you_want_your_project_created]
+cargo cunew [path_where_you_want_your_project_created]
     🤷   Project Name:
 ```
 
-Check out [copper-templates](https://github.com/copper-project/copper-rs/templates/README.md) for more info.
+Check out [copper-templates](templates/README.md) for more info.
```

Comment:
Ha I include the readme in the rust crate doc here: https://github.com/copper-project/copper-rs/blob/c0f6d8e1e18891a19a955f7fe7de3c4d5181711b/cu29/src/lib.rs#L1

It means that relative links won't work.

An alternative could be to move the templating into the cu29 crate?

Reply:
ah gotcha. Would this be a workaround https://gist.github.com/JakeHartnell/2c1fa387f185f5dc46c9429470a2e2be?

I can undo the relative links as well

Reply:
oooh yes, this is clever! let's try this trick

Reply:
fwiw, without changes, i was able to see the README when running `cargo doc` and opening up the results in firefox

I'll push the workaround since I don't know if it'll stay that way when published to https://copper-project.github.io/copper-rs/cu29/index.html

Reply:
Updated in 902afc9dce541477e55c40bc9afa2a87699c6ca1